src/main.py

```diff
@@ -217,6 +217,22 @@ async def image_view(interaction: Interaction) -> None:
     await interaction.response.send_message(content)
 
 
+# Info command
+@client.slash_command()
+@application_checks.has_role(config.dj_role_name)
+async def info(interaction: Interaction) -> None:
+    """Get info about currently listed songs."""
+    bc = controllers.get(interaction.guild_id)
+
+    if bc is None:
+        await interaction.response.send_message(
+            "You need to use /list first", ephemeral=True
+        )
+        return
```

Comment:
I wonder if we should automatically create a bust here / overwrite the current one with the latest information if `/list` has already been called. Otherwise we'll always need to `/list` manually first, or the info could be about an old, outdated `/list` invocation.

Reply:
If we give /info the same interface for passing in which channel to list I think it's a little confusing, in particular running `/info` with no arguments would depend on whether `/list` has been run for which channel it generates stats for. If something has been listed, then that channel. Otherwise, the one the message was sent in.

Maybe if you run with no arguments you MUST have already listed, but if you run with a channel argument you re-list that channel and then show info. I think it's a good idea, but I also think it should be a separate PR since it's basically just adding a feature on top of this one. If this sounds good to you we can make an issue and I'll get to it soonish

Reply:
Let's try it out in its current form and see whether it'd be a useful addition.